app/views/challenges/_header.html.erb

```diff
@@ -1,7 +1,7 @@
 <div id='challenge-header' class='<%= controller_name %>'>
   <div class="container">
     <div class="row">
-      <div class="col-sm-10 col-sm-offset-1">
+      <div class="col-md-12 col-sm-12 padding-horizontal-none">
```

Comment:
Hm, still trying to make sense of what's going on here with overriding horizontal padding. Not sure I follow what this change is meant / intended to do. It's one thing to get rid of the offsets, another to add the overrides. If we want this to take up the full container, then truly what we should do is both delete the `row` and `col` class (those are only needed when you plan on not taking up a full row, generally). The other reason you'd use a row / col with `col-XX-12` is if you *want* the left / right padding (which it seems in this case we don't want?).